content/en/docs/guides/certificate_management/certificate-rotation.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Root Certificate Rotation"
+description: "Rotating the root certificate in OSM"
+type: docs
+weight: 10
+---
+
+# Root Certificate Rotation
+
+The mesh root certificate is a long-lived certificate that is used by OSM to issue leaf certificates to services in the mesh thereby enabling mTLS. This document focuses on the steps to rotate a certificate, to learn more about OSM's certificates see [Certificate Provider Options](certificates.md).  
```

Comment:
I think this might lead me to think the MRC is only used for service certificates in the mesh. I think it also might be helpful to distinguish between the MRC and a certificate. The MRC references a certificate or indicates how to use the root certificate. It also contains other certificate settings.

Reply:
```suggestion
The mesh's root certificate is a long-lived certificate that is used by OSM to issue leaf certificates to services in the mesh thereby enabling mTLS. This document focuses on the steps to rotate a certificate, to learn more about OSM's certificates see [Certificate Provider Options](certificates.md).  
```

The intent of this paragraph was to root certificate not the MRC.  Would a change like the above make that clearer or should I make some more changes?